### PR TITLE
147 update user info in local storage

### PR DESCRIPTION
**Pull request by ervinj1998**
ACTUALIZACIONES EN EL PERFIL, y quitando el bug del loading cuando no habian datos en las casillas de datos

### CODE REVIEW

**Comment by Hawaiideveloper** [comment]
I will review this one in depth, as we will need to ensure it is meeting the requirements of our UI testing

**Comment by Hawaiideveloper** [comment]
as of present it has a merge conflict

File: src/user/EditProfile.js [hunk 1]
```diff
@@ -56,17 +56,20 @@ isValid = () => {
     }    
 
     if(name.length === 0){
-        this.setState({error: "Name is required"}) //validate name and prompt user validamos y condicionamos que el nombre no puede estar en blanco si quiere editar el perfil.
+        this.setState({error: "Name is required", loading: false }); //validate name and prompt user validamos y condicionamos que el nombre no puede estar en blanco si quiere editar el perfil.
```
Comment (manzanarez83) [inline src/user/EditProfile.js:59]: we added it: **loading: false});** in order to remove the word loading on the frontend, user will not see it anymore.

**Comment by Hawaiideveloper** [comment]
I have resolved the merge conflict.

Now merging

**Comment by Hawaiideveloper** [comment]
Screen Shot 2020-10-15 at 6 24 24 PM

It appears we have lost function to sign-up when a user is not found

**Comment by Hawaiideveloper** [comment]
this branch does not show the images anymore.  please review and let me know what happened

**Comment by Hawaiideveloper** [comment]
sorry...

I have found that you need a fresh session for new loads....

**Comment by Hawaiideveloper** [comment]
otherwise you will not see the images

**Comment by ervinj1998** [comment]
the type removes and adds the functions, currently that function if it is available, please if you change the code let me know to add it before making a new branch, or you will have to add those changes again

**Comment by Hawaiideveloper** [comment]
uhhhh.

yes it looses photos

File: src/user/apiUser.js [hunk 2]
```diff
@@ -69,4 +69,16 @@ export const read = (userId, token) => {
         return response.json();
     })
         .catch(err => console.log(err));
-        };
\ No newline at end of file
+
+};
+
+export const updateUser = (user, next) => {
```
Comment (manzanarez83) [inline src/user/apiUser.js:75]: **export const updateUser = (user, next) => {** in this method we expect to get to the user gets to the object to use information.

**Comment by Hawaiideveloper** [comment]
please explain again what has happened?

for local storage, it is bad form to load into master without testing a corresponding branch on a production ready system

**Comment by Hawaiideveloper** [comment]
please re-submit another PR @manzanarez83  please be sure it matches the local_storage chapter

**Pull request merged**